packages/config-loader/src/json/tiff.config.ts

```diff
@@ -442,16 +443,70 @@ export async function initConfigFromUrls(
     layers: [],
   };
 
+  const elevationTileSet: ConfigTileSetRaster = {
+    id: 'ts_elevation',
+    name: 'elevation',
+    title: 'Basemaps',
+    category: 'Basemaps',
```

Comment:
This could be `category: 'Elevation'`

Reply:
I am not sure what we want this to be, this area needs to be refactored more now that we are creating multiple tilesets. For instance should elevation datasets show in the "all" cateogry?